Final review of RFP docs

mheadd opened the pull request:
This pull request will add all of the RFP and related documents to the master branch, effectively releasing the RFP for the search tool.

Commit:
Added RFP and attachments for final review

randyhart committed:
Rename Request-for-Proposals.md to 1-Request-for-Proposals.md

randyhart committed:
Rename QAP.md to 2-QAP.md

randyhart committed:
Update 2-QAP.md

Commit:
Added How We Work document

Commit:
Merge branch 'final-review' of https://github.com/AlaskaDHSS/RFP-Search-Unification into final-review

Commit:
Removed "Working Draft" language from RFP, Fixed link to issues

randyhart committed:
Rename how-we-work.md to 3-how-we-work.md

randyhart committed:
Rename prototype-findings.md to 4-prototype-findings.md

randyhart committed:
Rename Attachment A - DHSS IT Requirements Agile Acquisition.md to 5-Attachment A - DHSS IT Requirements Agile Acquisition.md

randyhart committed:
Rename Attachment E - Person Service.wsdl to 7-Attachment E - Person Service.wsdl

randyhart committed:
updated links and other small edits

Commit:
Merge branch 'final-review' into randyhart-patch-4

Commit:
Merge pull request #9 from AlaskaDHSS/randyhart-patch-4

updated links and other small edits

Commit:
Update 1-Request-for-Proposals.md

Fixed link to prototype doc

Commit:
Renamed files

randyhart committed:
Update README.md

randyhart committed:
Update README.md

randyhart committed:
Update final QAP

fixed link to new RFP repo.

randyhart committed:
updated link to new RFP Repo

Commit:
Update 3-how-we-work.md

Fixed outdated links.

Commit:
Update 3-how-we-work.md

Fixed typo

Commit:
Update README.md

Added link to labels issues in project repo

Commit:
Update 5-Attachment A - DHSS IT Requirements Agile Acquisition.md

Removed redundant section from TOC and clean up.

Commit:
Update 2-QAP.md

Change EISR to EIS

randyhart reviewed (approved):
looks good - i did a scan of all of the links and they work.

sztaylorakgov committed:
Removed browser lifecycle analysis and upgrade req

Removed language that stated the Offeror must analyze browser lifecycle and include cost of upgrades, per conversation with Tracy Mack.

sztaylorakgov committed:
Fixed unintended typo/char removal

sztaylorakgov committed:
Removed irrelevant language 

Removed browser language from the desktop productivity software requirement.  Removed language from desktop productivity that required the Offeror to analyze lifecycle of productivity and include upgrade.

waldoj committed:
Change filename to mixed case

waldoj committed:
Change filename to mixed case

Commit:
Merge pull request #11 from AlaskaDHSS/fix-file-case

Fix file case
